src/messages/IPostMessageSentHandler.ts

```diff
@@ -1,4 +1,4 @@
-import { IExecutionResult, IHttp, IRead } from '../accessors';
+import { IExecutionResult, IHttp, IModify, IRead } from '../accessors';
```

Comment:
> we are going to just go off the description that is provided in the rocketlet.json file for displaying what the Rocketlet will do

There's nothing I could contradict more to. I sketched the analogy "Installing a Rocketlet" to "Installing an app on your smartphone". You want to know the responsibilities upfront. And you want to know them safely. I'd not install an app if it *can* e. g. read my emails without letting me know before installation.
One of the major USPs of Rocket.Chat compared to others is it's availability on premise with enterprise features for companies. I'd not install a single Rocketlet if I was unsure what it can do.